src/olympia/reviewers/views.py

```diff
@@ -1386,6 +1400,27 @@ def retrieve(self, request, *args, **kwargs):
         )
         return Response(serializer.data)
 
+    @action(
+        detail=True,
+        methods=['get', 'put', 'delete'],
+        permission_classes=ReviewAddonVersionMixin.permission_classes)
+    def draft_comment(self, request, **kwargs):
+        version = self.get_object()
+
+        if request.method == 'GET':
+            instance = get_object_or_404(DraftComment, version=version)
+            serializer = DraftCommentSerializer(instance, data=request.data)
+            return Response(serializer.data)
+        elif request.method == 'DELETE':
+            DraftComment.objects.filter(version=version).delete()
+            return Response(status=status.HTTP_202_ACCEPTED)
```

Comment:
Isn't `delete()` on a model synchronous? If so, shouldn't we return a 204?

Reply:
you're correct, but there may not be any `DraftComment` instance for that version to delete.  (should we throw different codes depending on if you try to `DELETE` the comment for a version when there aren't any?)

Reply:
I kinda think 204 is what I originally wanted. I don't think we need a different status code for different states, e.g an instance existing vs one doesn't. `204` mostly is just a success response.